tst/EnergyPlus/api/TestRuntime.py

```diff
@@ -62,6 +62,11 @@ def environment_handler(_state) -> None:
     sys.stdout.flush()
 
 
+def common_callback_handler(_state) -> None:
```

Comment:
Created a single callback handler, could always add more complexity if it was ever needed.